### DISCUSSION TITLE
How to run example project from Shoelace examples?

### DISCUSSION THREAD

**Discussion by checketts** [wimdeblauwe/wimdeblauwe.com#5] [Q&A] [answered]
I downloaded your code from the Shoelace blog posts but can't seem to get them to run. (https://github.com/wimdeblauwe/blog-example-code/tree/master/shoelace-thymeleaf-alpine)

I'm guessing there is a maven or npm task I need to have running but it isn't obvious to me what. Can you give me a hint?

Thanks,

-Clint

**Comment by wimdeblauwe** [comment, marked as answer]
You can run ‘mvn verify’ or you can run ‘npm run build’. After that, start
the application from IntelliJ.

Op za 18 mrt. 2023 om 05:12 schreef checketts ***@***.***>
